sidestep_engine/data/metadata_provider_music_flamingo.py

```
@@ -0,0 +1,1131 @@
+
+import ast
+import http.cookiejar
+import ipaddress
+import json
+import logging
+import os
+import re
+import uuid
+from itertools import islice
+from pathlib import Path
+from typing import Any, Dict, Optional, Tuple
```

Ruff (0.15.5)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)